fairscale/optim/oss.py

```diff
@@ -49,45 +49,58 @@ class OSS(Optimizer):
             optimizer to shard (default: SGD)
         group (group):
             torch.distributed group (default: group.WORLD)
+        broadcast_buffer_size (int):
+            the size of the buffer used to batch the small parameter tensors (default 1M).
     """
 
     #: The optimizer used for a given shard
     optim: Optimizer
 
     in_super_constructor: bool
 
-    def __init__(self, params: _params_t, optim: Type[Optimizer] = SGD, group: Optional[Any] = None, **default: Any):
+    def __init__(
```

Comment:
Please use 120 character length.

Reply:
I'm using everything the contribute guide is telling me to do (including editorconfig, pre-commit hook), can we fix that in the tooling ?

Reply:
I think it was folded by black because of the extra argument, I have the exact black version from ci, not much more that I can do